workflow/ignite/handlers/model_score.py

```diff
@@ -0,0 +1,73 @@
+import ignite
+from ignite.engine import Events
+from ignite.contrib.handlers.tensorboard_logger import OutputHandler
+
+from workflow.ignite.metrics import ReduceMetricsLambda
+from workflow.ignite.handlers.early_stopping import EarlyStopping
+from workflow.ignite.handlers.model_checkpoint import ModelCheckpoint
+from workflow.ignite.handlers.best_model_trigger import BestModelTrigger
+
+
+class ModelScore:
```

Comment:
I'm unsure about what to call this class/handler. It's essentially used to keep track of which iteration gave the best model. `BestModelTracker`? `ModelScoreTracker`? I can get behind `ModelScore` but maybe it's not really conveying the full use of the class?

Reply:
I also think the name is not 100%

I was thinking ModelScoreHandler originally but handler was removed because it is in the folder "handlers" and none of the others had a handler suffix

Reply:
Let's keep it as `ModelScore` for now then